packages/Confirmation/src/Confirmation.js

```diff
@@ -0,0 +1,147 @@
+import React from "react";
+import PropTypes from "prop-types";
+import Button from "@paprika/button";
+import { ShirtSizes } from "@paprika/helpers/lib/customPropTypes";
+import useI18n from "@paprika/l10n/lib/useI18n";
+import Popover from "@paprika/popover";
+import Trigger from "./components/Trigger";
+import contentStyles from "./ContentStyles.styles";
+
+import {
+  confirmStyles,
+  confirmHeaderStyles,
+  confirmDescriptionStyles,
+  confirmFooterStyles,
+} from "./Confirmation.styles";
+
+const propTypes = {
+  buttonSize: PropTypes.oneOf(ShirtSizes.DEFAULT),
+  confirmButtonType: PropTypes.oneOf(["primary", "destructive"]),
+  confirmLabel: PropTypes.node.isRequired,
+  description: PropTypes.node,
+  heading: PropTypes.node.isRequired,
+  defaultIsOpen: PropTypes.bool,
+  isPending: PropTypes.bool,
+  onCancel: PropTypes.func.isRequired,
+  onClose: PropTypes.func,
```

Comment:
This can be optional, for cases when this is used as an uncontrolled component.

Reply:
yeah it already isn't required

Reply:
🤦‍♂